thirdeye-ui/src/app/components/anomalies-create/preview-anomaly-chart/preview-anomaly-chart.component.tsx

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2022 StarTree Inc
+ *
+ * Licensed under the StarTree Community License (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at http://www.startree.ai/legal/startree-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT * WARRANTIES OF ANY KIND,
+ * either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { CardContent, Grid, Typography } from "@material-ui/core";
+import React, { FunctionComponent, useMemo } from "react";
+import { useTranslation } from "react-i18next";
+import { Anomaly } from "../../../rest/dto/anomaly.interfaces";
+import { DetectionEvaluation } from "../../../rest/dto/detection.interfaces";
+import { extractDetectionEvaluation } from "../../../utils/alerts/alerts.util";
+import { getUiAnomaly } from "../../../utils/anomalies/anomalies.util";
+import { AnomalyCard } from "../../entity-cards/anomaly-card/anomaly-card.component";
+import { TimeRangeButtonWithContext } from "../../time-range/time-range-button-with-context/time-range-button.component";
+import { AlertEvaluationTimeSeriesCard } from "../../visualizations/alert-evaluation-time-series-card/alert-evaluation-time-series-card.component";
+import { PreviewAnomalyChartProps } from "./preview-anomaly-chart.interfaces";
+
+export const PreviewAnomalyChart: FunctionComponent<PreviewAnomalyChartProps> =
+    ({
+        editableAnomaly,
+        timezone,
+        isLoading,
+        fetchAlertEvaluation,
+        evaluation,
+    }) => {
+        const { t } = useTranslation();
+
+        const uiAnomaly = useMemo(
+            () => getUiAnomaly(editableAnomaly as Anomaly),
+            [editableAnomaly]
+        );
+
+        const detectionEvaluation = useMemo<DetectionEvaluation | null>(() => {
+            if (!evaluation || !editableAnomaly) {
+                return null;
+            }
+
+            return {
+                ...extractDetectionEvaluation(evaluation)[0],
+                anomalies: [editableAnomaly as Anomaly],
+            };
+        }, [evaluation, editableAnomaly?.alert.id]);
+
+        return (
+            <>
+                <AlertEvaluationTimeSeriesCard
+                    disableNavigation
+                    alertEvaluationTimeSeriesHeight={500}
+                    anomalies={[editableAnomaly as Anomaly]}
+                    detectionEvaluation={detectionEvaluation}
+                    header={
+                        <CardContent>
+                            <Grid container justifyContent="space-between">
+                                <Grid item lg="auto" md="auto" sm={4} xs={12}>
```

Comment:
nice, it never occurred to me you can combine `auto` and set proportions with the different screen sizes all in one

Reply:
This isn't strictly necessary in our UI since we are't targeting multiple device widths for now (and the rest of the UI would break), but just wanted to keep this similar to the `AnomalyHeader` component's grid layout.